control-plane/tests/test_phase27_day2_runtime_contract.py

```
@@ -501,6 +501,123 @@ def assert_idp_outage_blocks(
 
+    def test_phase27_upgrade_rollback_uncertainty_freezes_authority_sensitive_progression(
+        self,
+    ) -> None:
+        store, service, promoted_case, _evidence_id, reviewed_at = (
+            self._build_phase19_in_scope_case()
+        )
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                host="127.0.0.1",
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)